avm/res/machine-learning-services/workspace/main.bicep

```diff
@@ -20,14 +20,23 @@ param location string = resourceGroup().location
 ])
 param sku string
 
-@sys.description('Required. The resource ID of the associated Storage Account.')
-param associatedStorageAccountResourceId string
+@sys.description('Optional. The type of Azure Machine Learning workspace to create.')
```

Comment:
Was there some guide I read that these should be in alphabetical order (I may have dreamed it!)

Reply:
Correct there is not rule. On the upside - the ReadMe Utility automatically groups and sorts the parameters ;)